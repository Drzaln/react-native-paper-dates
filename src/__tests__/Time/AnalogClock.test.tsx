import { render } from '@testing-library/react-native'

import AnalogClock from '../../Time/AnalogClock'

it('renders AnalogClock', () => {
  const { toJSON } = render(
    <AnalogClock
      hours={12}
      minutes={30}
      focused="hours"
      is24Hour={false}
      onChange={() => null}
    />
  )

  expect(toJSON()).toMatchSnapshot()
})
