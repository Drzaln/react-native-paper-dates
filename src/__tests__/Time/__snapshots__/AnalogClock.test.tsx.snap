// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AnalogClock 1`] = `null`;
